eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -357,55 +357,59 @@
 
     {% if hasContent and showMap %}
     {% set apiKey = ezpublish.configResolver.hasParameter('api_keys.google_maps') ? ezpublish.configResolver.getParameter('api_keys.google_maps'): "" %}
+    {% set latitude = latitude|default(0) %}
+    {% set longitude = longitude|default(0) %}
         <script>
             if (typeof(window.ezgmaplocationMapsScriptLoaded) == 'undefined') {
                 (function (win, doc) {
                     var myScript = document.createElement('script');
-                    myScript.src = '//maps.googleapis.com/maps/api/js{% if apiKey %}?key={{ apiKey }}{% endif %}';
```

Comment:
If this is not used anymore we should deprecate the setting (including describing that it is not used anymore)